atoMEC/writeoutput.py

```diff
@@ -396,11 +408,15 @@ def write_final_energies(energy):
         ent = energy.entropy
         ent_str = "{S:30s} : {S_x:10.4f}".format(S="Entropy", S_x=ent["tot"]) + spc
```

Comment:
I find this old style of f-strings rather unreadable. But we had already discussed this. Would be good, to switch at some point to the new style (best would be a dedicated PR) ...